### ISSUE TITLE
Switch to Swift testing

### ISSUE THREAD

**Issue by brentsimmons** [Ranchero-Software/NetNewsWire#4333]

**Comment by brentsimmons** [comment]
I have no plan yet. :)

**Comment by ankushkushwaha** [comment]
Oh okay, then we can postpone this.

**Comment by DandyLyons** [comment]
@ankushkushwaha , I would love to work on this issue. 

I recently added Swift Testing to 2 packages. 

Honestly, it shouldn't matter very much where we start since converting one test from XCTest to Swift Testing doesn't change the API, and therefore shouldn't break any other tests. 

But I think the Parser module is a good place to start, since it's a very well understood problem space with lots of resources.

**Comment by ankushkushwaha** [comment]
Okay, sounds interesting to me. @brentsimmons is the correct person to give permission for this :)

**Comment by brentsimmons** [comment]
It‘s going to be a while before we start this — we’re working on the 6.2 milestone and using Xcode 15.4 for now.

**Comment by DandyLyons** [comment]
Cool. @ me whenever you're ready. I'd love to work on this.

**Comment by brentsimmons** [comment]
I learned that you can’t actually debug with Swift tests — and debugging is critical. If this changes in the future, we can reopen this ticket.
